### ISSUE TITLE
Accessibility: graphic component image missing aria hidden

### ISSUE THREAD

**Issue by guywillis** [adaptlearning/adapt_framework#2295]
The image source in the graphic component is missing an aria-hidden attribute for when no there is no aria-label defined. Currently, it is written as:

```
<img src="{{_graphic.src}}" data-large="{{_graphic.large}}" data-small="{{_graphic.small}}" aria-label="{{_graphic.alt}}" />
```

Suggested solution:

```
<img class="graphic__image js-graphic-set-image-src" src="{{_graphic.src}}" data-large="{{_graphic.large}}" data-small="{{_graphic.small}}" {{#if _graphic.alt}}aria-label="{{_graphic.alt}}"{{else}}aria-hidden="true"{{/if}}>
```
